helmfile/values/velero-sc.yaml.gotmpl

```
@@ -93,6 +93,9 @@ schedules:
     schedule: {{ .Values.velero.schedule }}
     template:
       storageLocation: default
+      includedNamespaces:
+      {{ .Values.user.namespaces | toYaml | nindent 8 }}
+      {{ .Values.velero.includedNamespaces | toYaml | nindent 8 }}
```

We don't have user namespaces in sc.
```suggestion
      includedNamespaces:
      {{ .Values.velero.includedNamespaces | toYaml | nindent 8 }}
```

---

But I'm starting to question if we should even have this feature in sc. Right now we only backup things based on specific labels, this would conflict with that view.
We should discuss if it makes sense to add this for sc.

---

yep!